pkg/minishift/provisioner/utils.go

```diff
@@ -19,63 +19,83 @@ package provisioner
 
 import (
 	"fmt"
-	"github.com/docker/machine/libmachine/auth"
-	"github.com/docker/machine/libmachine/log"
 	"path"
 	"strings"
+
+	"github.com/docker/machine/libmachine/auth"
+	"github.com/docker/machine/libmachine/log"
 )
 
 var (
 	engineConfigTemplateRHEL = `[Unit]
 Description=Docker Application Container Engine
+Documentation=http://docs.docker.com
 After=network.target rc-local.service
 Requires=rhel-push-plugin.socket
+Requires=docker-cleanup.timer
 
 [Service]
 Type=notify
-ExecStart=/usr/bin/docker daemon -H tcp://0.0.0.0:{{.DockerPort}} -H unix:///var/run/docker.sock \
+NotifyAccess=all
+Environment=GOTRACEBACK=crash
```

Comment:
So how do you know what needs to change? Are you basically copying/updating from a default systemd setup for CentOS?

Have we been using a systemd config which was not in sync with the whatever should have been used from the CentOS side?